src/Contracts/Masa.Auth.Contracts.Admin/Subjects/UserSelectDto.cs

```diff
@@ -25,11 +25,13 @@ public UserSelectDto()
 
     public UserSelectDto(Guid id, string name, string account, string phoneNumber, string email, string avatar)
     {
-        Id = id;
+        Id = id;      
         Name = name;
         Account = account;
         PhoneNumber = phoneNumber;
         Email = email;
         Avatar = avatar;
+        Value = Id;
+        Text = $"{Name}{Account}{PhoneNumber}{Email}";
```

Comment:
Don't you need a delimiter?

Reply:
Like this kind of common can be put into SDK for everyone to use, or make a component.